src/main/java/dk/kb/netarchivesuite/solrwayback/properties/PropertiesLoaderWeb.java

```diff
@@ -87,8 +88,11 @@ public class PropertiesLoaderWeb {
 
     private static Properties serviceProperties = null;
     //Default values.
-    public static List<String> FACETS = Arrays.asList("domain", "content_type_norm", "type", "crawl_year", "status_code", "public_suffix"); 
+    public static List<String> FACETS = Arrays.asList("domain", "content_type_norm", "type", "crawl_year", "status_code", "public_suffix");
     public static String FIELDS=null;
+    public static List<String> STATS = Arrays.asList("content_length", "crawl_year", "content_text_length", "image_height", "image_width", "image_size",
```

Comment:
These should also be added to the sample property file `src/test/resources/properties/solrwaybackweb.properties`.